### DISCUSSION TITLE
Prevent accidental tab reset on new best

### DISCUSSION THREAD

**Discussion by KraXen72** [monkeytypegame/monkeytype#4520] [Ideas] [answered]
I'm so used to hitting tab right after a test, that when i get a new best, i subconsciously tab it away, and can't screenshot the screen. It would be nice if you could add an option that doesen't let you tab for 3/5s after a new best.

**Comment by Miodec** [comment, marked as answer]
Sure, why not, i can add a 3 second block

EDIT: Well, looking into this now, I don't know if a result is a pb until a response comes back form the server. The local data is not guaranteed to be 100% correct, so i just wait for the response. And then blocking the test restarting until the response arrives is also not the best UX, since the requests could get delayed. Not sure whats the best thing to do here.

**Reply by KraXen72** [reply]
add it as an option? (off by default)

**Reply by Miodec** [reply]
Its a bit too specific to be an option

**Reply by KraXen72** [reply]
okay, what if you allowed downloading the screenshot (like you do in the result screen) for the last 3/5 results? I know the screenshot thingy relies on the html elements being present and using something like html2canvas (i assume). Only thing that might be a problem is if the server does not store all the data like speed graph for past results.

**Reply by Miodec** [reply]
All the data is already available on the account page, and you can view the graph by clicking the graph icon

**Reply by KraXen72** [reply]
yeye but i want that sweet new best screenshot i forgot to take by tapping tab too fast

**Reply by Miodec** [reply]
Or you could... you know... not press tab as soon as possible?

**Reply by KraXen72** [reply]
it's hard coded in my muscle memory lol. yes i know it's my skill issue but 3s block is easier to implement that rewrite muscle memory i think
